compiler/rustc_codegen_llvm/src/lib.rs

```diff
@@ -9,6 +9,7 @@
 #![feature(crate_visibility_modifier)]
 #![feature(extern_types)]
 #![feature(nll)]
+#![feature(let_else)]
```

Comment:
It seems to have already been added by another PR in the meantime causing duplicate feature warnings :(
https://github.com/rust-lang/rust/pull/93218#issuecomment-1019379001